dbt/adapters/bigquery.py

```diff
@@ -316,7 +318,14 @@ def execute(cls, profile, sql, model_name=None, fetch=False, **kwargs):
 
     @classmethod
     def execute_and_fetch(cls, profile, sql, model_name, auto_begin=None):
-        return cls.execute(profile, sql, model_name, fetch=True)
+        status, res = cls.execute(profile, sql, model_name, fetch=True)
+        table = cls.get_table_from_response(res)
+        return res, table
```

Comment:
was this broken before? looks like the order of return values has changed. i would expect it to be `return status, table`

Reply:
@cmcarthur that is a typo -- good catch! unsure why the tests didn't fail....

Reply:
it turns out the only place this is called for bq is for tests, and that first return value is thrown away. Fixed regardless